frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.tsx

```diff
@@ -94,6 +94,9 @@ function ModelActionListItem({
         <div>
           <ActionTitle to={actionUrl}>{action.name}</ActionTitle>
           <ActionSubtitle>
+            {action.type === "implicit" && (
+              <ActionSubtitlePart>{t`Basic`}</ActionSubtitlePart>
+            )}
             {action.public_uuid && (
               <ActionSubtitlePart>{t`Public action form`}</ActionSubtitlePart>
             )}
```

Comment:
1. The mockup in #28534 says "Basic action" instead of "Basic". Is the change intentional?
2. Do we want to separately handle public basic actions? Right now it'd say something like "Basic · Public action form"; are we cool about that?

Reply:
@kulyk Changed to `Basic action`, I think it was an oversight from me. @mazameli could you take a look at the second question?

Reply:
1. From what I can tell, the copy indeed should be `Basic action`, and I see that Alexander already updated this.
2. I'll defer to @kdoh; he has way more context.

Reply:
Though fwiw this seems fine to me: 
<img width="447" alt="image" src="https://user-images.githubusercontent.com/2223916/231024178-5194af02-9066-4abe-acb3-faa07d91483f.png">